ask-watch.py

```diff
@@ -3,8 +3,35 @@
 import ask
 import os
 
+
 os.system('clear')
-print('⚡' + '\033[92m' + '️Ask' + '\033[0m' + ' Watcher')
+
+
+def style_string(text,color='', bold=False):
+
+	ret = text
+	if color == 'red':
+		ret = '\033[91m' + ret + '\033[0m'
+
+	if color == 'green':
+		ret = '\033[92m' + ret + '\033[0m'
```

Comment:
Instead of doing the same thing many times, create a dictionary with `91m`, `93m` and so on with *'red'* and *'yellow'*, etc. as keys. Then you can apply the styling (along with the changes from my previous comment about the `prefix` & `suffix` variable) like this: 

```python
colors = {
  'red': '91m',
  # add all colors like this...
  ...
}

return prefix + colors[color] + text + suffix
```